gallery/plot_transforms.py

```diff
@@ -178,3 +178,25 @@ def _plot(img, title, **kwargs):
 # performs gaussianblur transform on an image.
 gaus_blur_img = T.GaussianBlur(kernel_size=(5, 9), sigma=(0.4, 3.0))(orig_img)
 plot(gaus_blur_img, "Gaussian Blurred Image")
+
+
+####################################
+# AutoAugment
+# -----------
+# The :class:`~torchvision.transforms.AutoAugment` transform
+# AutoAugments data based on available AutoAugmentation Policies.
+# Use :class:`torchvision.transforms.AutoAugmentPolicy` to create policies.
+# Available  policies are IMAGENET, CIFAR10 and SVHN.
+
+cifar10_policy = T.AutoAugmentPolicy.CIFAR10
+imagenet_policy = T.AutoAugmentPolicy.IMAGENET
+svhn_policy = T.AutoAugmentPolicy.SVHN
+
+cifar_img = T.AutoAugment(cifar10_policy)(orig_img)
+imagenet_img = T.AutoAugment(imagenet_policy)(orig_img)
+svhn_img = T.AutoAugment(svhn_policy)(orig_img)
+
+
+plot(cifar_img, "Cifar10 Transformed Image")
+plot(imagenet_img, "Imagenet Transformed Image", with_orig=False)
+plot(svhn_img, "SVHN Transformed Image", with_orig=False)
```

Comment:
I'd suggest to illustrate more than one transform per policy, in order to showcase the variety of possible outcomes. We could maybe have a 3x5 grid with 5 images for each of the 3 policies?

Reply:
Sounds good, but the `plot` method does not support making a 3x5 grid, I'm pushing my code which is quite dirty and lengthy,

Also, this would make a lot of images? (If not arranged well in grid) If so we can move this to a separate file.